src/pages/publication/contributors_tab/components/SortableTable.tsx

```diff
@@ -12,12 +12,13 @@ import {
   Tooltip,
   TableContainer,
 } from '@material-ui/core';
-import { Field, FieldProps, FormikProps, useFormikContext } from 'formik';
+import { Field, FieldProps, FormikProps, useFormikContext, FieldArrayRenderProps, move } from 'formik';
```

Comment:
hvis du kan importere move her, betyr det at du kan importere replace, push og remove også?

Reply:
nei, de setter også verdien i formik. Tidligere gjorde vi det også slik med move, men da måtte vi kalle setField for hvert enkelt field, som trigger ny render hver sin nye render. Når vi importerer move her kan vi flytte elementet i lista uten at det lagres med en gang, så lagrer vi hele contributors en gang når alt er rett (sequence-verdier er oppdatert for hver contributor)

Reply:
eller, svaret er "ja", men da måtte vi manuelt kalt setFieldValue etterpå for hver operasjon også

Reply:
ok, skjønner 👍